### PR TITLE
Add M1 support

### PR DESCRIPTION
**Pull request by fearful-symmetry**
Right now, beats is breaking on M1/ARM because of how we divvy out the Darwin support across 64 and 32 bit darwin, meaning `Uptime` doesn't exist on ARM. This mostly shifts some files around.

### CODE REVIEW

**Comment by andresrc** [comment]
@elastic/agent @elastic/integrations

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add ARM support [#152]https://github.com/elastic/gosigar/pull/152
```
Comment (andrewkroh) [inline CHANGELOG.md:10]:
```suggestion
- Add darwin ARM support. [#152](https://github.com/elastic/gosigar/pull/152)
```

**Review by andrewkroh — APPROVED** [review]
LGTM 👍

**Comment by fearful-symmetry** [comment]
Alright, looks like there's some setup issues with CI...

**Comment by v1v** [comment]
No more CI issues

**Pull request merged**